Source/Core/Core/HW/WiimoteReal/IOWin.cpp

```diff
@@ -289,25 +388,34 @@ void WiimoteScanner::FindWiimotes(std::vector<Wiimote*> & found_wiimotes, Wiimot
 		detail_data = (PSP_DEVICE_INTERFACE_DETAIL_DATA)malloc(len);
 		detail_data->cbSize = sizeof(SP_DEVICE_INTERFACE_DETAIL_DATA);
 
+		SP_DEVINFO_DATA device_info_data;
+		ZeroMemory(&device_info_data, sizeof(SP_DEVINFO_DATA));
+		device_info_data.cbSize = sizeof(SP_DEVINFO_DATA);
+
 		// Query the data for this device
-		if (SetupDiGetDeviceInterfaceDetail(device_info, &device_data, detail_data, len, nullptr, nullptr))
+		if (SetupDiGetDeviceInterfaceDetail(device_info, &device_data, detail_data, len, nullptr, &device_info_data))
 		{
 			std::basic_string<TCHAR> device_path(detail_data->DevicePath);
-			Wiimote* wm = new WiimoteWindows(device_path);
-			bool real_wiimote = false, is_bb = false;
+			bool real_wiimote = false;
+			bool is_bb = false;
+			bool IsUsingToshibaStack = CheckForToshibaStack(device_info_data.DevInst);
 
-			CheckDeviceType(device_path, real_wiimote, is_bb);
-			if (is_bb)
-			{
-				found_board = wm;
-			}
-			else if (real_wiimote)
-			{
-				found_wiimotes.push_back(wm);
-			}
-			else
+			WinWriteMethod write_method = GetInitialWriteMethod(IsUsingToshibaStack);
+
+			CheckDeviceType(device_path, write_method, real_wiimote, is_bb);
+
+			if (real_wiimote)
 			{
-				delete wm;
+				Wiimote* wm = new WiimoteWindows(device_path, write_method);
+
+				if (is_bb)
+				{
+					found_board = wm;
+				}
+				else
+				{
+					found_wiimotes.push_back(wm);
+				}
```

Comment:
I don't mind having them. It's acceptable without, but not mandatory to be without.